klippy/extras/gcode_macro.py

```diff
@@ -181,6 +181,7 @@ def cmd(self, gcmd):
         kwparams.update(self.template.create_template_context())
         kwparams['params'] = gcmd.get_command_parameters()
         kwparams['rawparams'] = gcmd.get_raw_command_parameters()
+        kwparams['unixtime'] = calendar.timegm(time.gmtime())
```

Comment:
Python is not my forte, but can't this just be:
```suggestion
        kwparams['unixtime'] = time.time()
```

Reply:
That should actually be fine, yes. I was reading through [this](https://stackoverflow.com/questions/16755394/what-is-the-easiest-way-to-get-current-gmt-time-in-unix-timestamp-format) and there were some comments that stated that `time.time()` is not timezone aware and results different results in different timezones, but that seems to have been wrong information.

(Don't forget to also get remove the `calendar` import.)